lib/deployment-tasks.js

```diff
@@ -88,6 +88,12 @@ function buildImages(registry) {
 	return function(buildOptions) {
 		logger.info('Building Docker Images');
 		logger.info('Will build the following images:');
+
+		for(let name in buildOptions.images)
```

Comment:
`_.pickBy` is the equivalent of filter for objects. https://lodash.com/docs/4.17.4#pickBy